package.json

```diff
@@ -13,6 +13,7 @@
     "geo lookup"
   ],
   "scripts": {
+    "install": "scripts/install",
```

Comment:
Would you mind changing it to `post-install`? Otherwise there be no way to installs module dependencies since `npm install` be overridden

Reply:
changed to `postinstall`